src/SDKs/Azure.ApplicationModel.Configuration/data-plane/README.md

```diff
@@ -41,43 +78,98 @@ Properties of a Configuration Setting:
     IDictionary<string, string> Tags { get; set; }
 ```
 
-# Hello World
-The following example demonstrates how to initialize a ConfigurationClient and perform a basic operations in the Configuration Store.
+## Examples
+The following sections provide several code snippets covering some of the most common Configuration Service tasks, including:
+- [Create a Configuration Setting](#create-a-Configuration-Setting)
+- [Retrieve a Configuration Setting](#retrieve-a-Configuration-Setting)
+- [Update an existing Configuration Setting](#update-an-existing-Configuration-Setting)
+- [Delete a Configuration Setting](#delete-a-Configuration-Setting)
 
-To begin using ConfigurationClient, a connection string must be provided which specifies the configuration store endpoint and credentials to use when sending requests. This conneection string can be retrieved by the Azure Portal or by using the [Azure CLI](https://docs.microsoft.com/en-us/azure/azure-app-configuration/cli-samples). From that point, the example uses the client to set, retrieve and delete a configuration setting by its name.
+### Create a Configuration Setting
+Create a Configuration Setting to be stored in the Configuration Store.
+There are two (2) ways to store a Configuration Setting:
```

Comment:
Is the (2) for screen reader purposes?

Reply:
Not really. I guess I have that bad habit :(
I will remove the (2)